segmentation/utils/segmentation_utils.py

```diff
@@ -49,19 +49,17 @@ def compute_complete_expression_matrices(segmentation_labels, base_dir, tiff_dir
         # defined for mibitiff loading, we'll assume uniqueness for now but in the future we can address
```

Comment:
while I'm doing the de-verbosifying, I might reduce the line lengths in seg utils into pep8 compliancy

Reply:
I think they should all be passing already? Otherwise Travis would be mad. I set the max_line_length to 99 in the settings, so that's what its checking against

Reply:
idk, my linter says there are like 22 lines w/ >99 chars.  An example is line 81 w/ 147 characters

Reply:
Hmm, you're right. Hefty. For sure, go ahead and simplify